katas/adriel-gimenes/02-06/Tribonacci-Sequence/TribonacciSequence.cs

```diff
@@ -0,0 +1,14 @@
+namespace katas.AdrielGimenes;
+
+public class Xbonacci
+{
+  public static double[] Tribonacci(double[] signature, int n)
+  {
+    List<double> trib = signature.ToList();
+    for(int i = 3; i < n; i++)
+    {
+      trib.Add(trib[i-1] + trib[i-2] + trib[i-3]);
+    }
+    return trib.Skip(0).Take(n).ToArray();
```

Comment:
Is Skip(0) needed here? I think it doesn't really do anything with 0 as argumnet since it returns the same trib list, is that right?

Reply:
That's right, `Skip(0)` is not needed here